docs/build/tools/useful-tools.md

```diff
@@ -11,7 +11,7 @@ import TabItem from '@theme/TabItem';
 ---
 
 - [Chainweaver (Wallet & Workbench)](../../basics/chainweaver/chainweaver-user-guide/)
-- [Atom IDE](https://atom.io)
+- [Atom IDE](atom-ide)
```

Comment:
Is this line still necessary, when the link to Atom is in the toolbar itself and is distinct from the "Other" section?